crates/virtio-queue/src/state.rs

```diff
@@ -242,6 +242,10 @@ impl QueueStateT for QueueState {
         self.max_size
     }
 
+    fn size(&self) -> u16 {
```

Comment:
Just wondering, the reason for not adding this method to `Queue` and `QueueGuard` too is the upcoming change from https://github.com/rust-vmm/vm-virtio/pull/150?

Reply:
yes, to avoid confliction.